app/views/recipe_foods/index.html.erb

```diff
@@ -0,0 +1,15 @@
+<p style="color: green"><%= notice %></p>
+
+<h1 class="mb-10 text-3xl font-medium">Recipe foods</h1>
+
+<div id="recipe_foods">
+  <% @recipe_foods.each do |recipe_food| %>
+    <%= render recipe_food %>
+    <p>
+    <p class="my-7 border-solid w-56 rounded-md border-2 borde  r-slate-200 p-4 bg-gray-50 hover:bg-black hover:text-white text-center">
+      <%= link_to "Show this recipe food", user_recipe_recipe_foods_path %>
+    </p>
+  <% end %>
+</div>
```

Comment:
kindly ensure your design matches the design given, check the comparison below to see the differences 

Wireframe Design          |  Your design     
:-------------------------:|:-------------------------:
![image](https://user-images.githubusercontent.com/61408860/218188231-c5ba6aa6-b8b6-4829-b15a-8e66ddb88a87.png) |  ![image](https://user-images.githubusercontent.com/61408860/218188302-137131da-4e4b-48d7-b287-d8d58c2ac238.png)